prologin/concours/stechec/views.py

```diff
@@ -782,3 +785,13 @@ class ReportBugList(RedmineIssueListView):
         ('tracker_id', '=', 1),
         ('author_id', '=', 'me'),
     ]
+
+
+class LoginView(django.contrib.auth.views.LoginView):
```

Comment:
I don't think you need that, we already make the settings available to all the templates here: https://github.com/prologin/sadm/blob/82aa204b659b804fe6fc46164b9d41978c3331b8/prologin/concours/stechec/context_processors.py

You should just be able to use `settings` directly in the view.

Reply:
thanks, I'll change that :)